### ISSUE TITLE
Git continues to try to sync non-existant Computers

### ISSUE THREAD

**Issue by RockoTheeHut** [ironmansoftware/powershell-universal#3129]
### Version

4.2.12

### Severity

Low

### Steps to Reproduce

We upgraded servers recently. I turned off the old server, copied our SQL Database to a new server, and turned PSU on on the new server. New server is running as expected. I was able to remove the old server in computers. However, Git still shows the old servers. It looks like I can delete the entries in [dbo].[GitStatus] but this should probably be taken care of when a computer is removed.

### Expected behavior

```console
Git sync / gitstatus table is updated when a computer is removed
```

### Actual behavior

```console
Old computer still shows
```

### Additional Environment data

SQL 2022
Windows Server 2022
PSU 4.2.12
Single node

### Visuals

![image](https://github.com/ironmansoftware/issues/assets/136116901/ae6cc0ee-0857-4dd3-8cbe-89766e0b6576)
![image](https://github.com/ironmansoftware/issues/assets/136116901/8d2429b2-fd14-41e3-9132-3b7d2b3596ca)
![image](https://github.com/ironmansoftware/issues/assets/136116901/e5bcbb30-cd24-43e3-bbf7-8195cfd0a3c8)

**Comment by RockoTheeHut** [comment]
Had a server reboot during a git sync when our automated patching was done. There is now an entry stuck in "Sync Status" in the Processing state, Commit shows Running... with the Timestamp the time the server was rebooted. The server sync status is frozen as well.

I had to delete the phantom entry in the gitsettings table and make a small change for it to actually sync again. I did not know that a server was out of sync, there is no error otherwise.

**Comment by adamdriscoll** [comment]
We had an issue where the git sync would attempt to run (in addition to other internal jobs) on a schedule before the server was actually fully up and running causing weird issues like this one. It was resolved in 4.2.13. The issue with clearing out the deleted records will be resolved in v4.2.14.
